Kit/Widgets/Sensors.swift

```diff
@@ -180,7 +180,7 @@ public class SensorsWidget: WidgetWrapper {
         
         view.addArrangedSubview(selectSettingsRow(
             title: localizedString("Display mode"),
-            action: #selector(changeMode),
+            action: #selector(changeDisplayMode),
```

Comment:
What is this change for?

Reply:
This change only effects compiling on macOS 10.14.4 due to the described compile error.
It is not a problem on macOS 15 (as you note) nor on macOS 11
It is only a change because of clash of func names.

Reply:
Nope, no errors on macOS 10.15, 11, and 12.